### DISCUSSION TITLE
A way to run Supabase WITHOUT Docker

### DISCUSSION THREAD

**Discussion by cocktailpeanut** [supabase/supabase#20272] [Questions]
I understand that there are Supabase is a collection of many components like pgsql and you've simplified the setup by packaging them into a Docker image, but just wondering if there's an instruction on manually running those WITHOUT docker.

Just some steps to install the components and running them would be helpful.

I'm trying to find a way to package this in a different form factor that's easier and more lightweight to get started than Docker. I could infer by looking into the Dockerfile but think it would be nice to have a step by step install guide that does not use docker even if it's not for this case.

**Comment by awalias** [comment]
I think it would be a case of going to each of the repos and following the instructions for running each one, and compiling those instructions into a single document. It would definitely be useful for the community if you would be open to sharing any progress you make on this. 

For example you can quickly build gotrue (auth) using the go tooling and execute it like so: https://github.com/supabase/gotrue#quick-start
